docs/secure-connections.md

```diff
@@ -24,14 +24,14 @@ See our [Sauce Connect Proxy Security Overview](https://saucelabs.com/resources/
   <div className="box-wrapper" markdown="1">
     <div className="box box1 card">
       <div className="container">
-      <h3><a href="/secure-connections/sauce-connect">Using Sauce Connect Proxy</a></h3>
-      <p>Review the Sauce Connect Proxy system and network requirements, and then install the Sauce Connect Proxy client.</p>
+      <h3><a href="/secure-connections/sauce-connect">Sauce Connect Proxy Overview</a></h3>
+      <p>Learn about Sauce Connect Proxy system requirements and how to install the client.</p>
       </div>
     </div>
     <div className="box box2 card">
       <div className="container">
       <h3><a href="/secure-connections/sauce-connect/#setup-and-configuration">Setup and Configuration</a></h3>
-      <p>Find the right Sauce Connect Proxy tunnel configuration for you and your organization. Once you’re connected, use our code samples to run your first test.</p>
+      <p>Find the right Sauce Connect Proxy tunnel configuration for you and your organization.</p>
```

Comment:
^The description for the overview doesn't seem to align with the content in the linked topic -- the setup and configuration topic doesn't explicitly provide the info needed to determine which tunnel config to use (I found the info in this section: /secure-connections/#choosing-the-right-solution)